.github/workflows/tests.yml

```diff
@@ -11,14 +13,21 @@ jobs:
     if: github.repository == 'explosion/confection'
     strategy:
       fail-fast: false
+      # matrix:
```

Comment:
Turning off tests on every platform in favor of the smaller test matrix we use internally. This speeds up CI but tests here are quite fast anyway so leaving this as an open question for now.

Reply:
I think the CI runs rarely enough for repos like this so it's better to do a full sweep.

Since the changes don't affect the CI results, I'd rather not have these changes to the CI mixed into this PR?